extensions/amp-analytics/0.1/vendors.js

```diff
@@ -139,6 +139,33 @@ export const ANALYTICS_CONFIG = {
     }
   },
 
+  'mediametrie': {
+    'requests': {
+      'host': '//prof.estat.com/m/web',
+      'pageview': '${host}/${serial}?' +
+      'c=${level1}' +
```

Comment:
Please indent by two additional spaces all of the additional lines corresponding to the `'pageview'` definition.